src/pipecat/services/simli.py

```diff
@@ -11,19 +17,26 @@
 from pipecat.processors.frame_processor import FrameDirection, FrameProcessor, StartFrame
 
 import numpy as np
-from av import AudioFrame
-from av.audio.resampler import AudioResampler
 
-from simli import SimliClient, SimliConfig
 from loguru import logger
 
+try:
+    from av import AudioFrame
```

Comment:
Looks like `AudioFrame needs to be imported from `av.audio.frame`. My IDE was complaining otherwise.

```suggestion
    from av.audio.frame import AudioFrame
```

Reply:
Done